### PR TITLE
[ADD] product_template_tags_group

### PR DESCRIPTION
**Pull request by SirTakobi**
Allow to create a group for tags defined by https://github.com/OCA/product-attribute/tree/6e590f70a06dc6ddbd1bae7ce4fe430812c7d7b8/product_template_tags

### CODE REVIEW

File: product_template_tags_group/security/product_template_tag_group_rule.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <record id="product_template_tag_group_rule_company" model="ir.rule">
+        <field name="name">Multi-company for Product Tag Group</field>
+        <field name="model_id" ref="model_product_template_tag_group"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_create" eval="1"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_unlink" eval="1"/>
+        <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
+    </record>
+</odoo>
```
Comment (eLBati) [inline product_template_tags_group/security/product_template_tag_group_rule.xml:17]: I think a multi company rule is not need, as the main data are products, which have multi company rules.
I would follow the same approach as `product.category` about this

**Review by eLBati — CHANGES_REQUESTED** [review]

File: product_template_tags_group/security/product_template_tag_group_rule.xml (hunk 1 above)
Comment (eLBati) [reply product_template_tags_group/security/product_template_tag_group_rule.xml:17]: Nevermind, didn't see https://github.com/OCA/product-attribute/blob/3e94baf1757a5e9d783adf366f4a210660f661dc/product_template_tags/security/product_template_rule.xml

**Review by eLBati — APPROVED** [review]
Thanks

File: product_template_tags_group/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+manage_product_template_tag_group,manage_product_template_tag_group,model_product_template_tag_group,base.group_user,1,1,1,1
```
Comment (carmenbianca) [inline product_template_tags_group/security/ir.model.access.csv:2]: Comparing this against the permissions in `product`, the permissions should be `1,0,0,0` for `base.group_user` and `1,1,1,1` for `base.group_system`.

**Review by carmenbianca — CHANGES_REQUESTED** [review]
Mostly LGTM, one comment.

File: product_template_tags_group/security/ir.model.access.csv (hunk 2 above)
Comment (SirTakobi) [reply product_template_tags_group/security/ir.model.access.csv:2]: Thanks for the review!
Note that tags groups are not linked to products but only to tags, so these permissions have to be aligned with permissions defined on tags, and right now they are:
> ```
> <odoo>
>     <record id="product_template_tag_access_user" model="ir.model.access">
>         <field name="name">product.template.tag access user (in product_template_tags)</field>
>         <field name="model_id" ref="model_product_template_tag"/>
>         <field name="group_id" ref="base.group_user"/>
>         <field name="perm_read" eval="1"/>
>         <field name="perm_create" eval="1"/>
>         <field name="perm_write" eval="1"/>
>         <field name="perm_unlink" eval="1"/>
>     </record>
> </odoo>
> ```
(ref. https://github.com/OCA/product-attribute/blob/486f3e850158f88e8630a2fba5acda465967fa84/product_template_tags/security/product_template_tag.xml)

Then product tags are used in products (and that might be why you looked at product's permissions), but it looks like who defined product tags chose to use different permissions from products' permissions, here I am sticking with that choice.

Comment (carmenbianca) [reply product_template_tags_group/security/ir.model.access.csv:2]: Fair! Resolved.

**Review by carmenbianca — APPROVED** [review]

**Comment by SirTakobi** [comment]
@OCA/product-maintainers can this be merged? Thanks!

**Comment by SirTakobi** [comment]
@OCA/product-maintainers can this be merged? Thanks!

**Review by eLBati — APPROVED** [review]

**Comment by SirTakobi** [comment]
@OCA/product-maintainers can this be merged? It already has enough reviews, thanks!

**Comment by SirTakobi** [comment]
@OCA/product-maintainers can this be merged? It already has enough reviews, thanks!

**Pull request closed without merging**